packages/grafana-ui/src/slate-plugins/slate-prism/index.ts

```diff
@@ -158,3 +181,72 @@ function createDecoration({
 
   return myDec;
 }
+
```

Comment:
Something feels a bit messy about these next two functions, so I want to iterate on them a bit at a later point